internal/core/queries/get_ds_by_id.go

```diff
@@ -57,5 +59,16 @@ func (ch GetDeviceStyleByIDQueryHandler) Handle(ctx context.Context, query media
 		result.HardwareTemplateID = v.HardwareTemplateID.String
 	}
 
+	dd, err := ch.DDCache.GetDeviceDefinitionByID(ctx, result.DeviceDefinitionID)
+	if err != nil {
+		return nil, &exceptions.InternalError{
+			Err: fmt.Errorf("failed to get device definition"),
+		}
+	}
+
+	result.DeviceDefinition = coremodels.GetDeviceDefinitionStyleQueryResult{
```

Comment:
we'll need to find the powertrain attribute and modify it if the device style name matches any of the hybrid / plugin hybrid names